lib/ansible/modules/storage/glusterfs/gluster_volume.py

```diff
@@ -465,22 +539,31 @@ def main():
             new_bricks = []
             removed_bricks = []
             all_bricks = []
+            bricks_in_volume = volumes[volume_name]['bricks']
+
             for node in cluster:
                 for brick_path in brick_paths:
                     brick = '%s:%s' % (node, brick_path)
                     all_bricks.append(brick)
-                    if brick not in volumes[volume_name]['bricks']:
+                    if brick not in bricks_in_volume:
                         new_bricks.append(brick)
 
-            # this module does not yet remove bricks, but we check those anyways
-            for brick in volumes[volume_name]['bricks']:
-                if brick not in all_bricks:
-                    removed_bricks.append(brick)
+            if not new_bricks and len(all_bricks) < bricks_in_volume:
```

Comment:
Here is comparison of int and list: `len(all_bricks) < bricks_in_volume`.
`Ansible error: The error was: TypeError: '<' not supported between instances of 'int' and 'list'`

Reply:
@valentinsiryk please file a bug on that.